<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="XpsException.xml" source-language="en-US" target-language="zh-CN">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5726ec2e525182c1dc574a9f2f90f75f39be27501.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">726ec2e525182c1dc574a9f2f90f75f39be27501</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Xps.XpsException">
          <source>Serves as the base class for exceptions that are thrown by the XML Paper Specification (XPS) packaging and serialization APIs.</source>
          <target state="translated">用作由 XML 纸张规范 (XPS) 打包和序列化 API 所引发的异常的基类。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Windows.Xps.XpsException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error condition.</source>
          <target state="translated">一个描述错误情况的 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> class that provides a specific error condition.</source>
          <target state="translated">初始化提供特定错误情况的 <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> 类的一个新实例。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The data that is required to serialize or deserialize an object.</source>
          <target state="translated">序列化或反序列化对象时所需的数据。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The context, which includes source and destination, of the serialized stream.</source>
          <target state="translated">序列化流的上下文，包括源和目标。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> class that provides specific <ph id="ph2">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> and <ph id="ph3">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>.</source>
          <target state="translated">初始化提供特定 <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> 和 <ph id="ph2">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 的 <ph id="ph3">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> 类的一个新实例。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is protected.</source>
          <target state="translated">此构造函数受到保护。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.String,System.Exception)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error condition.</source>
          <target state="translated">一个描述错误情况的 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.String,System.Exception)">
          <source>The underlying error that caused the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph>.</source>
          <target state="translated">导致 <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> 的基本错误。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Windows.Xps.XpsException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> class that provides a specific error condition and includes the cause of the exception.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.Windows.Xps.XpsException" /&gt;</ph> 类的一个新实例，该类提供特定错误情况并包括引发异常的原因。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Windows.Xps.XpsException.#ctor(System.String,System.Exception)">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic), the current exception is thrown in a <ph id="ph4">`catch`</ph> block that handles the inner exception.</source>
          <target state="translated">如果<ph id="ph1">`innerException`</ph>参数不是<ph id="ph2">`null`</ph>(<ph id="ph3">`Nothing`</ph>在 Visual Basic 中) 中, 引发当前异常<ph id="ph4">`catch`</ph>处理内部异常的块。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>